src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/AuthenticationResponse.kt

```diff
@@ -17,4 +17,11 @@
  */
 package org.taktik.icure.services.external.rest.v1.dto
 
-class AuthenticationResponse(var healthcarePartyId: String? = null, var reason: String? = null, var successful: Boolean = false, var username: String? = null)
+class AuthenticationResponse(
+	var healthcarePartyId: String? = null,
+	var reason: String? = null,
```

Comment:
As the v2 will only be using the refreshToken, you can add a @Deprecated annotation on the fields that will be deleted